apply-ParagraphStandards.py

```diff
@@ -0,0 +1,149 @@
+                        if updated > finished:
+                            timediff = updated - finished
+                            diffminutes = divmod(timediff.total_seconds(), 60)[0]
+                            if tests == 'All' or tests == 'MDBlockGeneration':
+                                if diffminutes > 5:
+                                    print (notebook_id + ': paragraph #' + str(x) + ' was saved ' + str(diffminutes) + ' minutes after being generated')
+                                    print ('        Saved:  ' + str(updated))
+                                    print ('    Generated:  ' + str(finished))
+                    if tests == 'CodeBlockFix' or tests == 'All':
+                        if i['text'].startswith('%md') == False:
+                            print (notebook_id + ': paragraph #' + str(x) + ' is set to markdown editor yet does not start with %md')
+                        if 'enabled' in i['config']:
+                            if i['config']['enabled'] == True:
+                                print (notebook_id + ': paragraph #' + str(x) + ' has execution enabled')
+                                if modify == True:
+                                    i['config']['enabled'] = False
+                                    has_changes = True
+                        if 'status' in i:
+                            if i['status'] == "READY":
+                                print (notebook_id + ': paragraph #' + str(x) + ' status is READY, should be FINISHED')
+                                if modify == True:
+                                    i['status'] = 'FINISHED'
+                                    has_changes = True
+                        if 'editorHide' in i['config']:
+                            if i['config']['editorHide'] == False:
+                                print (notebook_id + ': paragraph #' + str(x) + ' Editor is NOT hidden')
+                                if modify == True:
+                                    i['config']['editorHide'] = True
+                                    has_changes = True
+                        # handle exceptions, apply these last
+                        if notebook_id in sanity_exceptions['Exceptions']:
+                            if i['id'] in sanity_exceptions['Exceptions'][notebook_id]:
+                                if 'MarkdownEditorOpen' in sanity_exceptions['Exceptions'][notebook_id][i['id']]:
+                                    if sanity_exceptions['Exceptions'][notebook_id][i['id']]['MarkdownEditorOpen'] == True:
+                                        i['config']['editorHide'] = False
+                                        i['config']['enabled'] = True
+                                        i['status'] = 'READY'
+                                        has_changes = True
+                else:
+                    # all other paragraph editors
+                    if tests == 'CodeBlockFix':
```

Comment:
You can join the lines above to eliminate a level of indentation: `else if tests == 'CodeBlockFix':`